packages/boost-demo/src/App.tsx

```diff
@@ -40,9 +42,11 @@ function App() {
   // If you're running this locally you'll need to override this value
```

Comment:
One nice thing about docker is that it includes a sample script that always generates the same  payload id. I should update the comment to acknowledge that.